docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,102 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). The hotplugged interface's model  will be `virtio` and
+it will be of type [bridge](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
+will be attached to the restarted VM. If these interfaces are persisted, they
+become part of the VM spec - i.e. will not be considered hotplugged.
+
+## Enabling network interface hotplug support
+Network interface hotplug support must be enabled via a
+[feature gate](https://kubevirt.io/user-guide/operations/activating_feature_gates/#how-to-activate-a-feature-gate).
+The feature gates array in the KubeVirt CR must feature `HotplugNICs`.
+
+## Requirements
+Adding an interface to a KubeVirt Virtual Machine requires first an interface
+to be added to a running pod. This is not trivial, and has some requirements:
+
+- [multus dynamic networks controller](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller):
+  this daemon will listen to annotation changes, and trigger multus to configure
+  a new attachment for this pod.
+- multus running as a [thick plugin](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/thick-plugin.md):
+  this multus version exposes an endpoint to create attachments for a given pod
+  on demand.
+
+## Adding an interface to a running VMI
+First start a VMI - e.g.
+[examples Fedora VMI](https://github.com/kubevirt/kubevirt/blob/main/examples/vmi-fedora.yaml).
```

Comment:
That is not a good example, it will not work outside kubevirtci due to the `image` set there. Mind sharing a full VM spec here, pointing to a public image hosted on quay? It would make this guide standalone, and we could reliably map between the `vmi-fedora` name used in the `addinterface` example and this

Reply:
Done, good catch.